### ISSUE TITLE
`test_send_queued_messages.py` recently started to time out

### ISSUE THREAD

**Issue by karlb** [raiden-network/raiden#4312]
The test used to run properly, but recently started to time out with the old settings. Significantly increasing the timeout made the test run (https://github.com/raiden-network/raiden/pull/4313), but I'm afraid that this might mask a real issue.

I tried bisecting for test_send_queued_messages since it seems to reliably fail for me when run by itself. Unfortunately, some commits were not able to execute tests, so the result is quite inconclusive:

```
There are only 'skip'ped commits left to test.                                 
The first bad commit could be any of:                                 
1bee3e8212f69bb812fb821ac7f5813a90f5ca26                                       
98e5b350cb3e037c883083cdd614ff947d6f6f0d       
23e10facc36d0d59433bdf9892e1939e46f1ef48     
5ed05840025973864a4a2981e097f828325f26f3       
7180e857d9b251c1af56021bc90e68c8af9682e0     
We cannot bisect more!    
```

**Comment by rakanalh** [comment]
Resolved with #4329

**Comment by rakanalh** [comment]
Re-opening this issue as the issue seems to come back.

**Comment by ulope** [comment]
#4333 should make many of the transport tests a lot more reliable

**Comment by rakanalh** [comment]
Example failure: 

https://circleci.com/gh/raiden-network/raiden/63494#tests/containers/16

[raiden-debug_2019-07-11T12:56:08.062912.log](https://github.com/raiden-network/raiden/files/3386030/raiden-debug_2019-07-11T12.56.08.062912.log)

**Comment by hackaugusto** [comment]
https://circleci.com/gh/raiden-network/raiden/64516#tests/containers/17

**Comment by ulope** [comment]
Seems to have been fixed with #4333.
